erpnext/patches/v10_0/item_barcode_childtable_migrate.py

```diff
@@ -9,7 +9,6 @@
 def execute():
     items_barcode = frappe.db.sql("SELECT name, barcode FROM tabItem WHERE barcode IS NOT NULL", as_dict=1)
     frappe.reload_doc("stock", "doctype", "item")
```

Comment:
Also reload `Item Barcode`

Reply:
ohh ..sorry ..tested on mine with both table in there :( ..fixed